static/app/components/group/inboxBadges/inboxReason.tsx

```diff
@@ -93,7 +93,7 @@ function InboxReason({inbox, fontSize = 'sm', showDateAdded}: Props) {
     tagType: React.ComponentProps<typeof Tag>['type'];
     reasonBadgeText: string;
     tooltipText?: string;
-    tooltipDescription?: string | React.ReactElement;
+    tooltipDescription?: string | React.ReactNode;
```

Comment:
are we sure about this change? `React.ReactNode` also brings other types like: `ReactFragment | ReactPortal | boolean | null | number`

Reply:
@priscilawebdev I think so. The reason is that we usually render all these as "children" which makes this closer to the signature of React.PropsWithChildren type

Reply:
in this case shouldn't we remove the type `string`?

Reply:
Yes, you are correct, we can drop that :)